test/fixtures/linter/autofix/ansi/016_index_error_with_jinja_if/after.sql

```diff
@@ -0,0 +1,16 @@
+-- This file combines product data from individual brands into a staging table
+{% set products =  [
+  'table1',
+  'table2'] %}
+
+{% for product in products %}
+SELECT
+    brand,
+    country_code,
+    category,
+    name,
+    id
+FROM
+    product
```

Comment:
This to me is wrong as `product` is a variable, so this should be:

```suggestion
    {{ product }}
```

However testing this leads to another problem (discussed below)...

Reply:
Oops, I just intended to use a placeholder table name. I'll change it to something else like `foo`.

Reply:
Actually I think we should use the real templated name (`{{ product }}`) since it has shown another error. Even if we don;'t fix that here. Again happy to commit my change if you want.